src/components/AppsMenu/components/AppItem.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import get from 'lodash/get'
+
+import { models } from 'cozy-client'
```

Comment:
does `import { applications } from 'cozy-client/dist/models'` have a tree shaking impact?

Reply:
It should be better. I updated it.